server/configmodule.py

```diff
@@ -14,7 +14,7 @@ class Config:
     VERSION = '{}-{}'.format(os.environ.get('WEBSITE_HASH'),
                              os.environ.get('MIXER_HASH'))
     API_ROOT = 'http://127.0.0.1:8081'  # Port for Kubernetes ESP.
-    GCS_BUCKET = os.environ.get('GCS_BUCKET') or ''
+    RECON_API_ROOT = 'http://127.0.0.1:8081'
```

Comment:
I see, so this is the instance of Recon+Mixer API service running co-located with Website?

Reply:
Yes, that's right.